### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by Ahmad-Ayyoub**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -71,9 +71,30 @@ The `system` and `inventory` services can be found at the following URLs:
...
+      }
+  ],
+  "outcome": "UP"
+}
+```
+
+As you can see, there are two checks. One shows the state of the `system` service, and the other shows the state of
 the `inventory` service. As you might expect, both services are in the `UP` state, and the overall health status of
-the application is `UP`.
+the application is `UP`. In each check, the data field is left empty for simplicity. This field can be
```
Comment (evelinec) [inline README.adoc]: I don't get why the field is left empty, is it cus we don't need it in the guide? "for simplicity" doesn't explain that.

File: README.adoc [hunk 2]
```diff
@@ -71,9 +71,30 @@ The `system` and `inventory` services can be found at the following URLs:
...
+filled with custom information regarding the health of the two services.
```
Comment (evelinec) [inline README.adoc]: You may want to re-phase "can be filled with" to use non passive voice. "This field can be filled with custom information regarding the health of the two services." doesn't sound good, and doesn't capture the description/purpose of the data field.

**Comment by Ahmad-Ayyoub** [comment]
Can @Kubik42 @JunyiSun @evelinec  please review the new changes ?

**Comment by yeekangc** [comment]
What is this PR for?  What issue(s) is it for?  @Ahmad-Ayyoub

**Comment by Ahmad-Ayyoub** [comment]
@yeekangc I believe it is a mix of these issues https://github.com/OpenLiberty/guide-microprofile-health/issues/41 & https://github.com/OpenLiberty/guide-microprofile-health/issues/40.

**Pull request closed without merging**